source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md

```diff
@@ -0,0 +1,310 @@
+---
+title: Up and Running with oVirt 3.6
+author: jbrooks
+date: 2016-02-29 14:00:00 UTC
```

Comment:
Please change the date to 2016-03-18 09:00:00 CET so that I can publish this tomorrow :)